DataConnectors/TrendMicroDeepSecurity.json

```diff
@@ -0,0 +1,134 @@
+{
+    "id": "TrendMicroDeepSecurity",
```

Comment:
Please do not change the id, else becomes two connectors that does the same thing. Going forward for newer connectors, please name IDs per the naming convention. Revert this to the old / existing ID

Reply:
I would have preferred to change this to correctly reflect the product this was for since MS chose the initial ID but as that isn't recommended I reverted it back.